src/libraries/System.Globalization/tests/System/Globalization/TextInfoTests.cs

```diff
@@ -245,6 +245,7 @@ public static IEnumerable<object[]> ToLower_TestData()
             }
         }
 
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/36672", TestPlatforms.Android)]
```

Comment:
Not a test.  Apply this attribute to the tests that call into `TestToLower`

Reply:
Fixed my script to put active issues above the test function definition, ignoring any calls of the test function